Blockzilla/AutocompleteCustomUrlViewController.swift

```diff
@@ -193,7 +193,7 @@ extension AutocompleteCustomUrlViewController: UITableViewDataSource {
         let itemToMove = domains[sourceIndexPath.row]
         _ = customAutocompleteSource.remove(at: sourceIndexPath.row)
         _ = customAutocompleteSource.add(suggestion: itemToMove, atIndex: destinationIndexPath.row)
-        Telemetry.default.recordEvent(category: TelemetryEventCategory.action, method: TelemetryEventMethod.customDomainReordered, object: TelemetryEventObject.setting)
+        Telemetry.default.recordEvent(category: TelemetryEventCategory.action, method: TelemetryEventMethod.customDomainReordered, object: TelemetryEventObject.customDomain)
```

Comment:
If this is a new probe, it's fine to change now since we haven't released it yet. @boek is this OK with you?